examples/components/index.html

```diff
@@ -168,6 +168,11 @@
             label: 'Date: Oldest first',
             sortBy: 'date',
             order: 'asc'
+          },
+          {
+            label: 'Most recently relevant',
+            sortBy: ['relevance', 'date'],
```

Comment:
This example won't work because currently search-api ignores `sort=relevance` and doesn't send it to search-front. search-front always automatically adds relevance as second sorting parameter when a different primary sorting is given.

If possible you should give an example with two fields that are not relevance.